__tests__/outdated-snapshot-reporter.spec.js

```diff
@@ -61,6 +61,7 @@ describe('OutdatedSnapshotReporter', () => {
       cwd: tmpDir,
       encoding: 'utf-8',
       env: { ...process.env, ...environment },
+      shell: true,
```

Comment:
https://nodejs.org/en/blog/vulnerability/april-2024-security-releases-2#command-injection-via-args-parameter-of-child_processspawn-without-shell-option-enabled-on-windows-cve-2024-27980---high